components/react/hooks/src/useScroll/index.js

```diff
@@ -0,0 +1,41 @@
+import {useState, useEffect} from 'react'
+
+export default function useScroll({axis = 'Y'} = {}) {
+  const getBoundingClientRect = () =>
+    window?.document ? document.body.getBoundingClientRect() : 0
```

Comment:
This is going to break SSR as window is not defined. Optional chaining could protect your from an `undefined` value but not a not defined variable.

Reply:
Ok, so is better with the first version , right?

👉🏼 https://github.com/SUI-Components/adevinta-spain-components/pull/191/files/413a52956fc2ef37bfdca1496c909d7fade0f141#diff-cd45a1a1bf87d161bc518751a0aaba1b

Reply:
Yes!. You could move outside the function a const like this:
```
const isClient = typeof window != 'undefined' && window.document
```

And later use here:
```
const getBoundingClientRect = () => isClient ? document.body. getBoundingClientRect() : 0
```